packages/apps/types/src/proto/schema.proto

```diff
@@ -27,3 +27,28 @@ message Meta {
 
   repeated ForeignKey keys = 1;
 }
+
+// TODO(burdon): Rename.
+// TODO(burdon): System type for external GUIDs (identifies resolver).
+// TODO(burdon): Timestamp, temporal context, TTL.
+// TODO(burdon): Dependency graph (resolvers that preceded this).
+message Source {
```

Comment:
i think we need to bring in @wittjosiah and @dmaretskyi to discuss how we evolve the type system.
also, i don't think stack.sections should be externalizable objects.

Reply:
This model did look like a useful way for the extensibility framework to find a distinct plugin for resolving the `object`… I was struggling to imagine how StackPlugin would resolve these on its own.

Reply:
@richburdon My assumption was that by default we'd want stacks not to own the objects referenced in their sections (i.e. soft links). This would mean that the links are just item ids I think? Unless you have some way to indicate that in the protobuf already